test-e2e/server-integration.js

```diff
@@ -201,14 +201,16 @@ test('observations endpoint', async (t) => {
 
       await project.$sync.waitForSync('full')
 
-      const response = await server.inject({
-        method: 'GET',
-        url: `/projects/${projectId}/observations`,
-        headers: { Authorization: 'Bearer ' + BEARER_TOKEN },
-      })
+      const response = await fetch(
```

Comment:
Mixing the server actually listening on a port and using `fastify.inject()` is confusing because the server behaves in different ways - should really change everything in this test to use fetch rather than inject, but that can be a follow-up.